zanata-war/src/main/java/org/zanata/rest/service/SourceDocResourceService.java

```diff
@@ -421,7 +421,7 @@ private HLocale validateTargetLocale(LocaleId locale, String projectSlug,
         } catch (ZanataServiceException e) {
             log.warn(
                     "Exception validating target locale {} in proj {} iter {}",
-                    e, locale, projectSlug, iterationSlug);
+                    locale, projectSlug, iterationSlug);
```

Comment:
Whether we should such an exception stacktrace could be debated, but I don't think it should change in this pull request.